public/app/features/dashboard-scene/serialization/transformSaveModelSchemaV2ToScene.ts

```diff
@@ -152,6 +152,8 @@ export function transformSaveModelSchemaV2ToScene(dto: DashboardWithAccessInfo<D
     dashboardNotFound: Boolean(dto.metadata.annotations?.[AnnoKeyDashboardNotFound]),
     version: parseInt(metadata.resourceVersion, 10),
     isNew: Boolean(dto.metadata.annotations?.[AnnoKeyDashboardIsNew]),
+
+    k8s: metadata,
```

Comment:
@dprokop I discovered a missing `k8s` metadata pass, resulting in the save form lacking access to metadata

Reply:
do we need it? Is the DashboardScene using that property? All the information this object has is already transformed to DashboardMeta

Reply:
It seems we do, checking the DashboardMeta type https://github.com/grafana/grafana/blob/main/public/app/types/dashboard.ts#L77 

```
  // When loaded from kubernetes, we stick the raw metadata here
  // yes weird, but this means all the editor structures can exist unchanged
  // until we use the resource as the main container
  k8s?: Partial<ObjectMeta>;
```

Reply:
I asked app platform, and they confirmed we need the whole k8s: [slack](https://raintank-corp.slack.com/archives/C07QVGH37D4/p1737637399514499)